docs/index.rst

```diff
@@ -76,13 +76,13 @@ Runtime systems
   .. image:: images/qiskit_runtime.png
       :scale: 50 %
       :align: center
-      :target: https://qiskit.org/documentation/partners/qiskit_runtime
+      :target: https://qiskit.org/documentation/partners/qiskit_ibm_runtime
 
 
   ++++++
   :link-badge:`https://quantum-computing.ibm.com/lab/docs/iql/runtime/,"Website",cls=badge-dark text-white`
```

Comment:
```suggestion
  :link-badge:`https://quantum-computing.ibm.com/lab/docs/iql/runtime/,"Legacy",cls=badge-dark text-white`
  :link-badge:`https://cloud.ibm.com/docs/quantum-computing?topic=quantum-computing-overview",cls=badge-dark text-white`
```

Reply:
I think we need more context here for this change. To the external user the difference between legacy and cloud is pretty opaque (especially as the ibm cloud interface is still in an experimental or pre-release phase right?) Just having little buttons on the bottom of the runtime picture that say "Legacy" and "Cloud" isn't super clear.

Reply:
Ok may be we can just merge without my suggestion for now. I will talk to documentation team on how better we can explain this.